tests/test_qubes_vms.py

```diff
@@ -30,7 +30,7 @@ def test_current_fedora_for_sys_vms(self):
             wanted_templates = [CURRENT_FEDORA_TEMPLATE]
             if sys_vm in sys_vms_maybe_disp:
                 if sys_vm in sys_vms_custom_disp:
-                    wanted_templates.append("sd-fedora-37-dvm")
+                    wanted_templates.append("sd-fedora-38-dvm")
```

Comment:
Shouldn't this also use CURRENT_FEDORA_VERSION?

Reply:
Probably, but this change can be deferred.